docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
+
+No degradation
+##############
+:Limitation: The MVS does not degrade the generation of a production asset over the lifetime of the project.
+:Reason: The MVS is based oemof-solph python library and uses its generic components to set up an energy system.
+:Implications: This results in an overestimation of the energy generated by the asset, which implies that the calculation of some other results might be unrealistic (e.g., overestimation of feed-in energy).
+
 Perfect foresight
 #################
-- Battery charge
-
-Infeasible dispatch of assets
-#############################
+:Limitation: The optimal solution of the energy system is based on perfect foresight.
+:Reason: As oemof-solph, as well as the MVS, know the generation and demand profiles for the whole simulation time and solve the optimization problem based on a linear equation system, the solver knows their dispatch for certain, whereas in reality the generation and demand could only be forecasted.
+:Implications: The perfect foresight can lead to suspicious dispatch of assets, for example charging of a battery right before a (in real-life) random blackout occurs. The systems optimized with the MVS therefore, represent their optimal potential, which in reality could not be reached. The MVS has thus a tendency to underestimate the needed battery capacity or the minimal state of charge for backup purposes, and also designs the PV system and backup power according to perfect forecasts. In reality, operational margins would need to be added.
 
-- Energy consumption and feed-in at the same time
-- Bi-directional inverters
+Extension of KPIs
+#################
+:Limitation: Some important KPIs, usually required by developers, are currently not implemented in the MVS.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: The absence of such indicators might affect decision-making.
+
+Random excess energy distribution
+#################################
+:Limitation: There is random excess distribution between the feed-in sink and the excess sink when no feed-in-tariff is assumed in the system.
+:Reason: Since there is no feed-in-tariff to benefit from, the MVS randomly distributes the excess energy between the feed-in and excess sinks.
+:Implications: This distribution of excess energy changes when running several simulations for the same input files.
+
+Renewable energy share defintion
+################################
+:Limitation: The current renewable energy share depends on the share of renewable energy production assets directly feeding the load.
```

Comment:
This does not describe the issue for me. You can throw in words like "energy carrier", "energy vector", the actual equation itself.